layers/meta-balena-genericx86/recipes-core/images/balena-image.bbappend

```diff
@@ -2,10 +2,16 @@ include balena-image.inc
 
 BALENA_BOOT_PARTITION_FILES_append = " \
 	grub.cfg_internal:/EFI/BOOT/grub.cfg \
+	grub.cfg_internal_luks:/EFI/BOOT/grub-luks.cfg \
```

Comment:
These is not only for this device type I guess. Other DTs that use a TPM based luks partition encryption would also need it, so what about moving it to meta-balena?

```
BALENA_BOOT_PARTITION_FILES:append = "${@oe.utils.conditional('SIGN_API','','',' grub.cfg_internal_luks:/EFI/BOOT/grub-luks.cfg',d)}"
```

Reply:
Moved to https://github.com/balena-os/meta-balena/pull/2880